kubernetes/assets/dashboards/kubernetes_pods.json

```diff
@@ -291,7 +291,7 @@
                 "type": "timeseries",
                 "requests": [
                     {
-                        "q": "sum:kubernetes.cpu.usage.total{$cluster,$namespace,$deployment,$scope,$statefulset,$daemonset,$job} by {kube_container_name,container_id}",
+                        "q": "sum:kubernetes.cpu.usage.total{$kube_cluster_name,$kube_namespace,$kube_deployment,$scope,$kube_stateful_set,$kube_daemon_set,$job} by {kube_container_name,container_id}",
```

Comment:
those that start with `kubernetes.` come from the kubelet integration which doesn't have this issue (we should still do `job --> kube_job` though)